src/api/grainConfig.js

```diff
@@ -4,23 +4,21 @@ import {type DistributionPolicy} from "../core/ledger/applyDistributions";
 import * as G from "../core/ledger/grain";
 import * as C from "../util/combo";
 import * as NullUtil from "../util/null";
-import {toDiscount} from "../core/ledger/policies/recent";
+import {
+  type AllocationPolicy,
+  allocationPolicyParser,
+} from "../core/ledger/policies";
 
 export type GrainConfig = {|
-  +immediatePerWeek?: number,
-  +balancedPerWeek?: number,
-  +recentPerWeek?: number,
-  +recentWeeklyDecayRate?: number,
+  +allocationPolicies: $ReadOnlyArray<AllocationPolicy>,
```

Comment:
Every AllocationPolicy has a budget field which is a raw attoGrain value, e.g. if they want to distribute 10000 Grain per period, they would need to write `10000000000000000000000`. This is not a user friendly UX.

As a proof-of-viability, in the next iteration please include in the PR description an example of what SC's own grain config file will look like once updated.

Reply:
Done, added parser to take number and transform it to `NonnegativeGrain` so users can write amounts as before.